lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/search/query/SearchFacetOption.json

```diff
@@ -0,0 +1,29 @@
+{
+	"description": "Specify which field should be returned as facets and the format of the results for each field",
+	"properties": {
+		"name": {
```

Comment:
make and enum and coordinate with Jay on the values (with an abstraction if possible).